pkg/apis/cluster/v1alpha1/cluster_types.go

```diff
@@ -35,6 +35,8 @@ import (
 // +kubebuilder:printcolumn:name="InfraType",type="string",JSONPath=".spec.infraType",description="Infra type of the cluster"
 // +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".spec.version",description="Kubernetes version of the cluster"
 // +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase",description="Phase of the cluster"
+// +kubebuilder:webhook:verbs=create;update,path=/validate-cluster-kurator-dev-v1alpha1-cluster,mutating=false,failurePolicy=fail,matchPolicy=Equivalent,groups=cluster.kurator.dev,resources=cluster,versions=v1alpha1,name=validation.cluster.cluster.kurator.dev,sideEffects=None,admissionReviewVersions=v1alpha1
+// +kubebuilder:webhook:verbs=create;update,path=/mutate-cluster-kurator-dev-v1alpha1-cluster,mutating=true,failurePolicy=fail,matchPolicy=Equivalent,groups=cluster.kurator.dev,resources=cluster,versions=v1alpha1,name=default.cluster.cluster.kurator.dev,sideEffects=None,admissionReviewVersions=v1alpha1
```

Comment:
can you change the path here

Reply:
let me find if it's possible to change this.

https://github.com/kubernetes-sigs/controller-runtime/blob/9e4e84418cd0cdf8d10e4430d1d66fc92b9ef81c/pkg/builder/webhook.go#L242